### PR TITLE
[WFLY-6607] Broadcast/Discovery group is possible to create with just a name

### PR DESCRIPTION
**Pull request by bayern39**
https://issues.jboss.org/browse/WFLY-6607

### CODE REVIEW

**Comment by bayern39** [comment]
retest this please

File: messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/DiscoveryGroupAdd.java [hunk 1]
```diff
@@ -62,6 +65,25 @@ private DiscoveryGroupAdd() {
     }
 
     @Override
+    protected void populateModel(OperationContext context, ModelNode operation, Resource resource) throws OperationFailedException {
+        super.populateModel(context, operation, resource);
+
+        final PathAddress address = PathAddress.pathAddress(operation.require(OP_ADDR));
+        final String name = address.getLastElement().getValue();
```
Comment (bstansberry) [inline messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/DiscoveryGroupAdd.java]: The 2 lines should be:

String name = context.getCurrentAddressValue();

Same effect but less code and uses some cached data inside the context so a bit more efficient.

File: messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/logging/MessagingLogger.java [hunk 2]
```diff
@@ -819,4 +819,6 @@
     @Message(id = 87, value = "Unable to load connector service factory class: %s")
     OperationFailedException unableToLoadConnectorServiceFactoryClass(String factroyClass);
 
+    @Message(id = 88, value = "neither socket-binding nor jgroups-channel are defined in %s")
```
Comment (bstansberry) [inline messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/logging/MessagingLogger.java]: "neither" should be capitalized.

File: messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/logging/MessagingLogger.java [hunk 3]
```diff
@@ -819,4 +819,6 @@
...
+    OperationFailedException notSocektbindingAndJgroupschannel(String name);
```
Comment (bstansberry) [inline messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/logging/MessagingLogger.java]: There's a typo in this method name.

**Comment by bstansberry** [comment]
I have put some detail comments above, but those are based on the assumption that specific validation of this combination is required. It really shouldn't be required though; it should be possible to declare that both attributes do not allow null, and that they are alternatives to each other. With that the standard model validation provided by the kernel will kick in and no custom code is needed.

However, I believe point 3) that I discuss in the description of https://issues.jboss.org/browse/WFCORE-1556 impedes this. That issue should be fixable though in time for WildFly 11 and EAP 7.1, so if this is not urgently needed (e.g. has been triaged for an EAP 7.0 CP release) then I think it may be better to wait for WFCORE-1566 and just mark these attributes as not allowing null. Otherwise if we merge this now we should have an issue to revert it later and do the simple fix.

**Comment by bayern39** [comment]
@bstansberry   Hi Brian,this is not urgently needed,I will close this pull request and wait for the WFCORE-1556 ,so can do the simple fix.

**Pull request closed without merging**